src/compat/posix/fs/dvfs/fcntl.c

```diff
@@ -0,0 +1,26 @@
+/**
+ * @file fcntl.c
+ * @brief Implementation of the POSIX fcntl function
+ * @date 3 Apr 2015
+ * @author Denis Deryugin
+ */
+
+#include <fcntl.h>
+#include <kernel/task/resource/file_table.h>
+
+int fcntl(int fd, int cmd, ...) {
+	va_list args;
+	union {
```

Comment:
It was originally copy-pasted from old fcntl() and then adopted to new VFS.
I agree, it's bad. Will be fixed